core/src/main/java/io/aiven/klaw/service/SchemaOverviewService.java

```diff
@@ -322,4 +333,19 @@ private boolean verifyIfTopicExistsInTargetSchemaEnv(
     String kafkaEnvId = promotedEnv.getId();
     return kafkaEnvIds.contains(kafkaEnvId) && promotedEnv.getAssociatedEnv() != null;
   }
+
+  private boolean isSchemaPromoteRequestOpen(String topicName, String envId, int tenantId) {
+    Optional<String> schemaEnvId =
+        manageDatabase.getAssociatedSchemaEnvIdFromTopicId(envId, tenantId);
+    return schemaEnvId.isPresent()
+        ? manageDatabase
+            .getHandleDbRequests()
+            .existsSchemaRequest(
+                topicName,
+                RequestStatus.CREATED.value,
+                RequestOperationType.CREATE.value,
+                schemaEnvId.get(),
```

Comment:
@muralibasani This is one thing i noticed promote schema requests are created with a CREATE requestOperationType, I can change this in this PR or I can do this in a follow up PR.

Reply:
its ok in this pr, as its related to promote